### PR TITLE
Add talk for R/Pharma APAC track

### PR DESCRIPTION
**Pull request by danielinteractive**

### CODE REVIEW

File: slides/rpharma-apac-2024.qmd [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+title: "Introducing openstatsware"
+subtitle: "Who we are and what we build together"
+author: "Daniel Sabanés Bové on behalf of `openstatsware`"
+date: "2024/02/23"
```
Comment (ellessenne) [inline slides/rpharma-apac-2024.qmd:5]: Is this date correct?

**Review by ellessenne — CHANGES_REQUESTED** [review]
Looks good! 😃 

I have a question, should this be added in the "Presentations" page as well to provide a link to slides straight away?

File: slides/rpharma-apac-2024.qmd (hunk 1 above)
Comment (danielinteractive) [reply slides/rpharma-apac-2024.qmd:5]: No, thanks! :-)

**Review by ya-wang-git — APPROVED** [review]

**Comment by ya-wang-git** [comment]
Thanks Daniel. Looks good!

**Pull request merged**